src/main/scala/sbtcatalysts/CatalystsPlugin.scala

```diff
@@ -393,6 +392,15 @@ trait CatalystsBase {
       }
     }
   )
+ 
+  lazy val partialUnificatonSettings = Seq(
```

Comment:
Any reason you don't use [sbt-partial-unification](https://github.com/fiadliel/sbt-partial-unification)?  That handles Scala 2.10 with the plugin.  Just trying to avoid an extra dependency?

Reply:
yes. avoiding extra dependency, as well as avoiding using the si-2712 compiler plugin fix by default.